src/libaktualizr/primary/aktualizr.h

```diff
@@ -71,6 +74,15 @@ class Aktualizr {
     if (api_thread_.joinable()) {
       api_thread_.join();
     }
+    if (config_.pacman.force_reboot) {
+      // TODO: put rebooting code in a correct place. I suggest to introduce Aktualizr::Deinitialize as symetric
+      // operations to Initialize() that actually initializes the api thread
+      // This code can go down to the specific package manager so it decides actually what exactly to do, reboot or just
+      // remove the sentinel file as in case of a fake manager
+      LOG_INFO << "About to reboot the system...";
+      sync();
+      reboot(LINUX_REBOOT_CMD_RESTART);
+    }
```

Comment:
I think this code actually belongs in the specific package managers, and it should get called through the package manger interface automatically at the end of installation, presumably after the installation report is sent.

Reply:
I feel like this code should go to a specific package manager too, will do it in the next commit.
In terms of where the code should be called from I am not sure what is the right place. Based on my understanding of the source code/design I think Actualizr dtor is the right place since here we can be sure that other threads are terminated ( like the API thread or the ReportQueue thread ) and don't do anything that should not be terminated by a reboot call (e.g. rebooting in the middle of writing to DB)